spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/SparkTableUtil.java

```diff
@@ -678,11 +678,9 @@ public static List<SparkPartition> filterPartitions(
   }
 
   private static void deleteManifests(FileIO io, List<ManifestFile> manifests) {
-    Tasks.foreach(manifests)
+    FileIOUtil.bulkDeleteManifests(io, manifests)
```

Comment:
Looks like we dont pass a name here.  Thinking about it, if a name is always recommended, why not make it mandatory parameter  of the bulkDelete API?